js/index.js

```diff
@@ -33,51 +35,122 @@ function renderEverything() {
     } else {
-      onePep.style.visibility = 'hidden';
+      onePep.style.visibility = "hidden";
+      document.querySelector(".btn-pepperoni").classList.add("active");
     }
   });
 }
 
 function renderMushrooms() {
+  document.querySelectorAll(".mushroom").forEach(oneMushrooms => {
+    if (state.mushrooms) {
+      oneMushrooms.style.visibility = "visible";
+      document.querySelector(".btn-mushrooms").classList.remove("active");
+    } else {
+      oneMushrooms.style.visibility = "hidden";
+      document.querySelector(".btn-mushrooms").classList.add("active");
+    }
+  });
   // Iteration 1: set the visibility of `<section class="mushroom">`
 }
 
 function renderGreenPeppers() {
-  // Iteration 1: set the visibility of `<section class="green-pepper">`
+  document.querySelectorAll(".green-pepper").forEach(oneGrepep => {
+    if (state.greenPeppers) {
+      oneGrepep.style.visibility = "visible";
+      document.querySelector(".btn-green-peppers").classList.remove("active");
+    } else {
+      oneGrepep.style.visibility = "hidden";
+      document.querySelector(".btn-green-peppers").classList.add("active");
+    }
+  });
 }
 
 function renderWhiteSauce() {
-  // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">`
+  document.querySelectorAll(".sauce").forEach(oneSau => {
+    if (state.whiteSauce) {
+      oneSau.classList.add("sauce-white");
+      document.querySelector(".btn-sauce").classList.remove("active");
+      // const sauprice = document.createElement("li");
+      // sauprice.innerHTML = "$3 white sauce";
+      // const parent = document.querySelector("body > aside > ul");
+      // parent.appendChild(sautprice);
+    } else {
+      oneSau.classList.remove("sauce-white");
+      document.querySelector(".btn-sauce").classList.add("active");
+      // document
+      //   .querySelector("body > aside > ul")
+      //   .removeChild(
+      //     document.querySelector("body > aside > ul > li:nth-child(4)")
+      //   );
+    }
+  });
 }
 
 function renderGlutenFreeCrust() {
-  // Iteration 2: add/remove the class "crust-gluten-free" of `<section class="crust">`
+  document.querySelectorAll(".crust").forEach(oneGlu => {
+    if (state.glutenFreeCrust) {
+      oneGlu.classList.remove("crust-gluten-free");
+      document.querySelector(".btn-crust").classList.remove("active");
+
+      // const crutprice = document.createElement("li");
+      // crutprice.innerHTML = "$5 gluten-free crust";
+      // const parent = document.querySelector("body > aside > ul");
+      // parent.appendChild(crutprice);
+    } else {
+      oneGlu.classList.add("crust-gluten-free");
+      document.querySelector(".btn-crust").classList.add("active");
+      // document
+      //   .querySelector("body > aside > ul")
+      //   .removeChild(
+      //     document.querySelector("body > aside > ul > li:nth-child(5)")
+      //   );
+    }
+  });
 }
```

Comment:
On this both functions you could have used querySelector because there is just one element .sauce and one .crust.

function renderWhiteSauce() { const oneSauce = document.querySelector(".sauce"); if (state.whiteSauce) { oneSauce.classList.add("sauce-white"); } else { oneSauce.classList.remove("sauce-white"); } }

function renderGlutenFreeCrust() { const oneCrust = document.querySelector(".crust"); if (state.glutenFreeCrust) { oneCrust.classList.add("crust-gluten-free"); } else { oneCrust.classList.remove("crust-gluten-free"); } }